roles/bootstrap-os/tasks/main.yml

```diff
@@ -8,4 +8,12 @@
 - include: bootstrap-centos.yml
   when: bootstrap_os == "centos"
 
-- include: setup-pipelining.yml
\ No newline at end of file
+- include: setup-pipelining.yml
+
+- name: check if atomic host
+  stat:
+    path: /run/ostree-booted
+  register: ostree
+
+- set_fact:
+    is_atomic_host: "{{ ostree.stat.exists }}"
```

Comment:
Can we just name it as 'is_atomic'? That way is default in many projects. Thanks